xarray/backends/common.py

```diff
@@ -37,6 +57,64 @@ def _normalize_path(path):
     return path
 
 
+def _find_absolute_paths(
+    paths: str | NestedSequence[str | os.PathLike], **kwargs
```

Comment:
```suggestion
    paths: str | os.PathLike | NestedSequence[str | os.PathLike], **kwargs
```

Reply:
Actually, why did you type this as a nested sequence? If I understand the code correctly only a "normal" sequence should be supported (even though it won't crash, just not parse anything).